examples/react-app/architect.yml

```diff
@@ -33,6 +38,7 @@ services:
     image: postgres:11
     interfaces:
       postgres:
+        host: ${{ parameters.api_db_host }}
```

Comment:
you can set username/password here. it'll be included in the ${{ services.api-db.interfaces.postgres.url }}

```yaml
services:
  api-db:
    image: postgres:11
    interfaces:
      postgres:
        port: 5432
        protocol: postgres
        username: ${{ parameters.root_db_user }}
        password: ${{ parameters.root_db_pass }}
        # path: ${{ parameters.api_db_name }} I'll add a ticket to support path since we should support
    environment:
      POSTGRES_USER: ${{ parameters.root_db_user }}
      POSTGRES_PASSWORD: ${{ parameters.root_db_pass }}
      POSTGRES_DB: ${{ parameters.api_db_name }}

...

api:
    environment:
      DB_ADDR: ${{ services.api-db.interfaces.postgres.url }}/${{ parameters.api_db_name }}
```

Then the typeorm config can be:
```ts
url: process.env.DB_ADDR
```